index.js

```diff
@@ -41,6 +41,7 @@ class JestReportPortal {
     constructor(globalConfig, options) {
         const agentInfo = getAgentInfo();
         this.reportOptions = getClientInitObject(getOptions.options(options));
+        if(this.reportOptions.enabled ==='false') return {invalid:true};
```

Comment:
Could you explain this line?
What is "invalid: true" here, does it have some specification?